user_sync/config.py

```diff
@@ -356,6 +357,7 @@ def get_rule_options(self):
         options = self.options
         result = {
             'directory_group_filter': options['directory_group_filter'],
+            'directory_group_mapped': options['directory_group_mapped'],
```

Comment:
The RuleProcessor shouldn't need to know about the directory_group_mapped or not.   The app could pass the right directory_group_filter before RuleProcessor is created.